src/coveralls.erl

```diff
@@ -338,10 +345,40 @@ replce_char_test_() ->
   , ?_assertEqual("foobarfoo", replace_char("foo\"foo", $", "bar")) %"
   ].
 
-fullpath_test_() ->
-  [ ?_assertEqual("/a/b", fullpath(["/", "a", "b"], []))
-  , ?_assertEqual("a/c", fullpath(["a", "b", "..", ".", "c"], []))
-  , ?_assertEqual("/", fullpath(["..", ".", "/"], []))
+expand_test_() ->
+  [ ?_assertEqual("/a/b", expand(["/", "a", "b"], []))
+  , ?_assertEqual("a/c", expand(["a", "b", "..", ".", "c"], []))
+  , ?_assertEqual("/", expand(["..", ".", "/"], []))
+  ].
+
+realpath_and_relative_test_() ->
```

Comment:
This seems odd. Could it just be the case that the tmp directory is called something else on travid? mktemp -d (without any path) should give a directory in the temp folder (whatever that is). If this is not the case I will look into it when I have time (maybe during the weekend).